src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private final Random random = new Random();
+
+    public String getRandomColor() {
+        int index = random.nextInt(Color.values().length);
+        return Color.values()[index].name();
```

Comment:
Consider storing the result of `Color.values()` in a local variable before using it in the `nextInt` and indexing, as calling `values()` multiple times can be less efficient. Each call to `values()` creates a new array.